src/main/java/com/smartcodeltd/jenkinsci/plugins/buildmonitor/Config.java

```diff
@@ -0,0 +1,37 @@
+package com.smartcodeltd.jenkinsci.plugins.buildmonitor;
+
+import com.smartcodeltd.jenkinsci.plugins.buildmonitor.order.ByName;
+import hudson.model.AbstractProject;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+import java.util.Comparator;
+
+public class Config {
+
+  private Comparator<AbstractProject> order;
+
+  public Comparator<AbstractProject> getOrder() {
+    return order;
```

Comment:
I can't imagine a worse usecase here than the new ordering doesn't take effect until the next (5 second) refresh. Can you?  It's not like the comparators have any intrinsic state themselves either.